src/schemas/21-527/schema.js

```diff
@@ -7,10 +7,80 @@ let schema = {
   title: 'INCOME, NET WORTH, AND EMPLOYMENT STATEMENT',
   type: 'object',
   additionalProperties: false,
-  definitions: _.pick(definitions,
-    'dateRange',
-    'netWorthAccount'
-  ),
+  definitions: _.merge(_.pick(definitions,
+    'dateRange'
+  ), {
+    netWorth: {
+      type: 'object',
+      properties: {
+        bank: { type: 'integer' },
```

Comment:
in this 527EZ which looks like the latest has them separated https://www.vba.va.gov/pubs/forms/VBA-21-527EZ-ARE.pdf